assignments/prototypes.js

```diff
@@ -102,7 +142,7 @@
   console.log(archer.greet()); // Lilith offers a greeting in Elvish.
   console.log(mage.takeDamage()); // Bruce took damage.
   console.log(swordsman.destroy()); // Sir Mustachio was removed from the game.
-*/
+
 
   // Stretch task: 
```

Comment:
Try to do this stretch, it's a fun stretch, as well as it will give you some good skills